database/sugar-rush.sql

```diff
@@ -0,0 +1,91 @@
+-- --------------------------------------------------------
+-- Hôte:                         127.0.0.1
+-- Version du serveur:           8.0.33 - MySQL Community Server - GPL
+-- SE du serveur:                Win64
+-- HeidiSQL Version:             12.5.0.6677
+-- --------------------------------------------------------
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET NAMES utf8 */;
+/*!50503 SET NAMES utf8mb4 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+-- Listage de la structure de la base pour sugar-rush
+CREATE DATABASE IF NOT EXISTS `sugar-rush` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
+USE `sugar-rush`;
+
+-- Listage de la structure de la table sugar-rush. article
+CREATE TABLE IF NOT EXISTS `article` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `author_id` int NOT NULL,
+  `category_id` int NOT NULL,
+  `leads` varchar(255) DEFAULT NULL,
+  `content` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci,
+  `is_main` bit(1) NOT NULL,
+  `modification_date` datetime(6) DEFAULT NULL,
+  `publication_date` datetime(6) DEFAULT NULL,
+  `publication_image` varchar(255) DEFAULT NULL,
+  `slug` varchar(255) DEFAULT NULL,
+  `title` varchar(255) DEFAULT NULL,
+  `auth` int NOT NULL,
+  `categ` int NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `FK92kabsgi6rs0pmr2866m7q7m9` (`author_id`),
+  KEY `FKy5kkohbk00g0w88fi05k2hcw` (`category_id`),
+  CONSTRAINT `FK92kabsgi6rs0pmr2866m7q7m9` FOREIGN KEY (`author_id`) REFERENCES `author` (`id`),
+  CONSTRAINT `FKy5kkohbk00g0w88fi05k2hcw` FOREIGN KEY (`category_id`) REFERENCES `category` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=9 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+
+-- Listage des données de la table sugar-rush.article : ~8 rows (environ)
+INSERT INTO `article` (`id`, `author_id`, `category_id`, `leads`, `content`, `is_main`, `modification_date`, `publication_date`, `publication_image`, `slug`, `title`, `auth`, `categ`) VALUES
```

Comment:
Change author_id to author, and the corresponding value to string (that is temporary because we don't have the users yet). auth and categ doesn't exist on the article model. Delete them and their corresponding values. In the content value, can you remove the balise (opening and closing) <section>